app.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+(function(){
+  angular
+  .module("todo", [
+    "xeditable"
+  ])
+  .controller("todo_controller", TodoController);
+
+  function TodoController() {
+    var vm = this;
+    vm.data = data;
+
+    vm.destroy = function(todo){
+      var index = vm.data.indexOf(todo);
+      vm.data.splice(index, 1);
+    };
+
+    vm.new_todo = {};
+    vm.create = function(){
```

Comment:
It's totally fine the way you have it now! But hopefully this at least gives you another idea or way to solve it